KubeArmor/config/config.go

```diff
@@ -157,6 +159,8 @@ func readCmdLineParams() {
 
 	throttleSec := flag.Int(ConfigThrottleSec, 30, "Time period for which subsequent alerts will be dropped (in sec)")
 
+	annotateResources := flag.Bool(ConfigAnnotateResources, true, "for kubearmor deployment without kubearmor-controller")
```

Comment:
```suggestion
	annotateResources := flag.Bool(ConfigAnnotateResources, false, "for kubearmor deployment without kubearmor-controller")
```
We keep this to false by default, since controller will handle historical containers as well!